### PR TITLE
Ticket Tailor: UI Review Revisions

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

Follow on design review from this [PR](https://github.com/guardian/support-frontend/pull/6210)...

Spacing adjustments and footer copy removal

[**Figma**](https://www.figma.com/design/E4KGImPY7IiIyEn8yKfDh4/GLive-events?node-id=346-4926&t=uKPLYvl5YRXHnmRQ-1)

[**Trello Card**](https://trello.com/c/915GAuRd/965-ticket-tailor-ui-refinement)

## How to test
https://support.thegulocal.com/uk/events?eventId=4452337&chk=296a

## Screenshots

|Mobile|Desktop|
|-----|-----|
|image|image|

### CODE REVIEW

File: support-frontend/assets/pages/[countryGroupId]/events/events.tsx [hunk 1]
```diff
@@ -64,7 +62,12 @@ export function Events() {
 			header={<Header />}
 			footer={
 				<FooterWithContents>
-					<ThankYouFooter />
+					<ThankYouFooter
+						copyOverride="&nbsp;"
+						cssOverrides={css`
+							margin-bottom: ${space[3]}px;
+						`}
+					/>
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/[countryGroupId]/events/events.tsx:70]: Should we just remove this from the events page and create a new element here?

**Review by jamesgorrie — APPROVED** [review]
Happy to merge this in - but will probably refactor to not use the `ThankYouFooter` on the events page.

File: support-frontend/assets/pages/[countryGroupId]/events/events.tsx (hunk 1 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/[countryGroupId]/events/events.tsx]: Agree, too complicated, have removed it completely as you suggest, thanks.

**Comment by paul-daniel-dempsey** [comment]
> This works - but will probably refactor to not use the `ThankYouFooter` on the events page.

Agree so removed it

**Comment by jamesgorrie** [comment]
> > This works - but will probably refactor to not use the `ThankYouFooter` on the events page.
> 
> Agree so removed it

Really nice simplification.

**Pull request merged**